openerp/http.py

```diff
@@ -1254,6 +1258,23 @@ def start_wrapped(status, headers):
             start_response(status, new_headers)
         return self.app(environ, start_wrapped)
 
+class OpenERPSessionStore(werkzeug.contrib.sessions.FilesystemSessionStore):
```

Comment:
Instead of this i would prefer adding thins at line 1393:
       if httprequest.session.rotate:
            self.session_store.delete(httprequest.session)
            httprequest.session.sid = httprequest.session.generate_key()
            httprequest.session.modified = True 

This makes no assumption on the implementation.